src/simsopt/geo/curvexyzfourier.py

```diff
@@ -72,16 +74,77 @@ def set_dofs(self, dofs):
         sopp.CurveXYZFourier.set_dofs(self, dofs)
 
     @staticmethod
-    def load_curves_from_file(filename, order=None, ppp=20, delimiter=','):
+    def load_curves_from_file(filename: str, order: int, ppp=20, delimiter=',', maxiter=2000, tol=1.48e-8):
```

Comment:
What is the change introduced in this function? If it is to support makegrid format, use a different function and leave the existing functionality alone.

Reply:
To keep the number of functions to a minimum I adapted the already existing function "load_curves_from_file" to also support the makegrid format instead of creating a new one, lets say "load_coils_from_makegrid_file". Would it be better to have two separate functions?